src/metabase/sync/sync_metadata/fks.clj

```diff
@@ -92,28 +92,6 @@
                                                                 :schema (:fk-table-schema metadata))
                               (sync-util/field-name-for-logging :name (:pk-column-name metadata)))))))
 
-(mu/defn sync-fks-for-db!
```

Comment:
This is now inlined in `sync-fks!` [below](https://github.com/metabase/metabase/pull/41588/files#diff-e09c063342055336a262f66623a6a1783a89871e4426c33c92f8f9b5f2bcaaecR110). We didn't need a separate function here.